samples/boards/nrf52/sensor_temperature/src/main.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <device.h>
+#include <gpio.h>
+#include <sensor.h>
+
+#define LED_PORT	DT_GPIO_LEDS_LED0_GPIO_CONTROLLER
+#define LED0		DT_GPIO_LEDS_LED0_GPIO_PIN
+#define LED1		DT_GPIO_LEDS_LED1_GPIO_PIN
+#define LED2		DT_GPIO_LEDS_LED2_GPIO_PIN
+#define LED3		DT_GPIO_LEDS_LED3_GPIO_PIN
+int led_write_values[4] = {0};
+
+/* 1000 msec = 1 sec */
+#define SLEEP_TIME	1000
+
+#define TEMPERATURE_READ_FAILED  1
+
+int get_new_temperature_reading(
+	struct device *temperature_dev,
+	struct sensor_value *val_ptr)
+{
+	if (sensor_sample_fetch_chan(temperature_dev, SENSOR_CHAN_DIE_TEMP)) {
+		printk("sensor fetch failed.\n");
+		return TEMPERATURE_READ_FAILED;
+	}
+
+	if (sensor_channel_get(temperature_dev,
+			       SENSOR_CHAN_DIE_TEMP,
+			       val_ptr)) {
+		printk("sensor channel get failed.\n");
+		return TEMPERATURE_READ_FAILED;
+	}
+
+	return 0;
+}
+
+void main(void)
+{
+	struct device *dev;
+	struct device *nrf_temperature_dev;
+	struct sensor_value temp_sensor_val;
+
+	printk("nRF52-DK Temperature Sensor Demo.\n");
+
+	/* Get LED Driver binding and set LED pins as output */
+	dev = device_get_binding(LED_PORT);
+	gpio_pin_configure(dev, LED0, GPIO_DIR_OUT);
+	gpio_pin_configure(dev, LED1, GPIO_DIR_OUT);
+	gpio_pin_configure(dev, LED2, GPIO_DIR_OUT);
+	gpio_pin_configure(dev, LED3, GPIO_DIR_OUT);
+
+	/* Get NRF5 Temperature Driver binding */
+	nrf_temperature_dev = device_get_binding(CONFIG_TEMP_NRF5_NAME);
+	if (!nrf_temperature_dev) {
+		printk("Cannot find %s!\n", CONFIG_TEMP_NRF5_NAME);
+		return;
+	}
+
+	while (1) {
+		/* Read sensor value and print to console */
+		if (get_new_temperature_reading(nrf_temperature_dev,
+						&temp_sensor_val)) {
+			printk("ERROR: Sensor value read failed. \n");
+			return;
+		}
+		printk("sensor value: %d, %d.\n",
```

Comment:
Maybe mention the temp is in degrees Celsius:
```suggestion
		printk("sensor value: %d C, %d C.\n",
```

Reply:
(Y)